live-examples/css-examples/lists/counter-increment.html

```diff
@@ -0,0 +1,35 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="list-style">
```

Comment:
`list-style` -> `counter-increment`

Reply:
Oops, the hazards of copying and pasting. I'll fix this in _all_ the places I made this particular mistake.